packages/utils/src/open-search/re-populate.ts

```diff
@@ -120,32 +120,9 @@ function isFromCommonWell(doc: DocumentReference) {
   if (!cw) return false;
   return true;
 }
-function isMetriportExtension(e: Extension): boolean {
-  return e.valueCoding?.code === METRIPORT;
-}
-function isCommonwellExtension(e: Extension): boolean {
-  return (
-    e.valueReference?.reference === COMMONWELL || // Legacy FHIR resources have this
-    e.valueCoding?.code === COMMONWELL
-  );
-}
 function hasMetriportAttachment(doc: DocumentReference) {
   return doc.content?.some(isMetriportContent) === true;
 }
-function isMetriportContent(content: DocumentReferenceContent): boolean {
-  // Metriport is the fallback/default.
-  // All doc refs created before this extension was added will have only one content element,
-  // stored on S3 (Metriport) and w/o the extension.
-  // So, return true if it's explicitly Metriport or is not explicitly CommonWell.
-  return content.extension?.some(isMetriportExtension) === true || !isCommonwellContent(content);
-}
-function isCommonwellContent(content: DocumentReferenceContent): boolean {
-  return (
-    content.extension?.some(isCommonwellExtension) === true ||
-    content.attachment?.url?.includes("commonwellalliance.org") || // Legacy FHIR resources only have this
-    false
-  );
-}
```

Comment:
is all this stuff unused now? I don't see where `isCommonwellExtension` or `isCommonwellContent` is defined

Reply:
Yea its unused since now its being imported (and used in the imported file)